### DISCUSSION TITLE
is it possible to make a slash menu with beautiful-mentions

### DISCUSSION THREAD

**Discussion by dodoto** [sodenn/lexical-beautiful-mentions#586] [Q&A]
is it possible to make a slash menu with beautiful-mentions

**Comment by sodenn** [comment]
Hi @dodoto. You don't need the `lexical-beautiful-mentions` plugin for this. Use `LexicalTypeaheadMenuPlugin` instead: https://github.com/facebook/lexical/issues/2114
